misago/admin/views/generic/list.py

```diff
@@ -134,7 +134,7 @@ def dispatch(
             if request.GET.get("set_filters"):
                 # Force store filters in session
                 session_key = self.filters_session_key
-                request.session[session_key] = context["active_filters"]
+                request.session[session_key] = context["active_filters"] or {}
```

Comment:
Why is this line changed too? Looks like bad place to fix `context` because few lines down we have other item access on `context` that will possibly fail if context is not a dict outside of this condition.

Reply:
You're right.